spec/shared_examples.rb

```diff
@@ -29,6 +29,19 @@
     end
   end
 
+  context 'Prepare' do
+    it 'should prepare resource' do
+      res = @manager.prepare(@resource['id'])
+      expect(res.instance_variable_get(:@entity)).to be_nil
```

Comment:
It is really not recommended to test instance variables. Tests should test public API - accessors rather than instance variables.